randutil/crypto_test.go

```diff
@@ -1,7 +1,7 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
This is slightly unexpected behavior of git/github to show it as a rename (?). Anyway, nothing wrong with it, just scratching my head.

Reply:
yes, confusing, it's really git though, I saw it in my local status before committing as well, it's git rename auto-detection being silly